src/style.css

```diff
@@ -0,0 +1,139 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  font-family: Avenir, sans-serif;
```

Comment:
it's better to move this into `html`

Reply:
I tried to move "font-family" into "html", but it didn't work

Reply:
move to html and apply font-family: inherit; this should work